src/Facilitator.ts

```diff
@@ -70,18 +80,38 @@ export default class Facilitator {
    *
    * @return <any> Object containing chain based subscriptionQueries.
    */
-  public static getSubscriptionDetails(): any {
+  public static getSubscriptionDetails(): SubscriptionInfo {
     return {
       origin: {
-        subGraphEndPoint: 'ws://localhost:8000/subgraphs/name/openst/ost-composer',
+        wsSubGraphEndPoint: 'ws://localhost:8000/subgraphs/name/openst/ost-composer',
+        httpSubGraphEndPoint: 'http://localhost:8000/subgraphs/name/openst/ost-composer',
         subscriptionQueries: {
-          stakeRequested: 'subscription{stakeRequesteds{id amount'
-          + ' beneficiary gasLimit gasPrice gateway nonce staker stakeRequestHash}}',
+          stakeRequested: 'subscription{stakeRequesteds(orderDirection: desc, first: 1){id}}',
```

Comment:
We can also fetch `contractAddress` from subscription data. 

As the db table we plan to create for keeping track of last processed timestamp for each entity would also have contract address as a column, we should also fetch `contractAddress` from subscription data. Without which we may have problems specially for `stateRootAvailable` (which would with redeem flow implementation have two rows in this table, one for origin and other for aux anchor).  

`stakeRequested: 'subscription{stakeRequesteds(orderDirection: desc, first: 1){id,contractAddress}}',`

Reply:
Done!